public/content/documentation/native/notifications/alert-dialog.md

```diff
@@ -0,0 +1,313 @@
+
+   - **UIKit** 
+
+      - When the focus lands on the alert's title, it is implied to the user that the alert is open.
+
+   - **SwiftUI**
+
+      - When the focus lands on the alert's title, it is implied to the user that the alert is open.
+
+### Focus
+
+   - Use the default focus functionality of the alert
+   - The screen reader focus **must** be confined within the alert. When the alert appears, the initial focus should be to a logical place or to where the default focus is for the device within the alert.
+
+   - **UIKit**
+
+      - If needed, follow the below for focus management.
+      - If VoiceOver is not reaching a particular element, set the element's `isAccessibilityElement` to `true`
+         - **Note:** You may need to adjust the programmatic name, role, state, and/or value after doing this, as this action may overwrite previously configured accessibility.
+      - To move screen reader focus to newly revealed content, use `UIAccessibility.post(notification:argument:)` that takes in `.screenChanged` and the newly revealed content as the parameter arguments.
+      - To NOT move focus, but dynamically announce new content: use `UIAccessibility.post(notification:argument:)` that takes in `.announcement` and the announcement text as the parameter arguments.
+      - `UIAccessibilityContainer` protocol: Have a table of elements that defines the reading order of the elements. 
+
+   - **SwiftUI**
+
+      - If needed, follow the below for focus management.
+      - For general focus management that impacts both screen readers and non-screen readers, use the property wrapper `@FocusState` to assign an identity of a focus state.
+         - Use the property wrapper `@FocusState` in conjunction with the view modifier `focused(_:)` to assign focus on a view with `@FocusState` as the source of truth.
+         - Use the property wrapper `@FocusState`in conjunction with the view modifier `focused(_:equals:)` to assign focus on a view, when the view is equal to a specific value.
+      - If necessary, use property wrapper `@AccessibilityFocusState` to assign identifiers to specific views to manually shift focus from one view to another as the user interacts with the screen with VoiceOver on.
+
+### Announcement examples
+
+   - Three separate focus areas
+      - "Alert, title" (When alert appears with an audible ping)
+      - "Body content"
+      - "CTA, button"
+
+## Android Developer Notes
+### General Notes
+
+   - An alert gives users ciritcal information they need right away, usually appearing over an existing screen
+   - Use native alerts when at all possible vs a custom element, as it will handle expected behavior without additional development effort
+   - Options to close the modal for the screen reader user: 
+      - Two/three finger swipe
+      - A close button
+      - Call To Action (CTA)
+      - Tapping outside the alert to close **cannot** be the only option for screen reader users
+
+### Name
```

Comment:
Seems like there's an extra space after between bullets within the Name, Role, Groupings, and State sections (vs. the General Notes section).